db/models.py

```diff
@@ -1 +1,37 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE
```

Comment:
Add `related_name` here as well

Reply:
Initially, I added it, but got this error after running pytest -> https://postimg.cc/YhvSHMmQ
Pytest works fine without it